recipes/parsmurf/meta.yaml

```diff
@@ -0,0 +1,66 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "parSMURF" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/AnacletoLAB/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 426013313b22d8fdd0db907e920e869347fb53e9e25cd59f07adc4b2eaf51ad7
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('cxx') }}
+    - cmake >=2.8
+    - openmpi
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - numpy ==1.13.0
+    - scipy ==1.2.1
+    - weave ==0.17.0
+    - six ==1.12.0
+    - protobuf ==3.7.1
+    - openmpi
+  run:
+    - numpy ==1.13.0
+    - scipy ==1.2.1
+    - weave ==0.17.0
+    - six ==1.12.0
+    - protobuf ==3.7.1
+    - openmpi
+    
+test:
+  files:
+    - parSMURF1
+    - parSMURFn
+  commands:
+    - parSMURF1 --help
+
+about:
+  home: https://github.com/AnacletoLAB/parSMURF
+  license: GPL-3.0
+  license_family: GPL
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#license-file
+  license_file: LICENSE
```

Comment:
From the README, it seem that this is vendoring things?

https://github.com/AnacletoLAB/parSMURF#license

If so, these should used as dependencies and the other packages should be added to conda-forge as needed. Iff there is a very good reason not to, the licenses must be included here.

Reply:
Oh this is tricky because the developer decided to download and install these packages during the install process. So bringing them to conda-forge will not change anything because the installation process will be the same..

How Do I include the license? extra files?

Reply:
I added the licencse file

Reply:
@chrisburr I think you have to resolve the review request